src/ui/window/ShortcutManager.qml

```diff
@@ -3,9 +3,19 @@ import core 1.0
 import ".."
 
 Item {
+    id: shortcutManager
+
+    property var root
     property TabsModel tabsModel
     property Toolbar toolbar
     property TabBar tabBar
+    property SearchOverlay searchOverlay
+
+    // when declaring BrowserWindow as property
+    // application fails to load for some reasons
+    // so find in page shortcut will be shared as a signal
+    // as a workaround
+    signal findInPage
```

Comment:
You can just use `searchOverlay.open` instead, as you defined a property for it anyway.